02_TrackWeb/src/utils.ts

```
@@ -0,0 +1,79 @@
+    return `https://${url}`;
+  }
+}
+
+/**
+ * Formats time in seconds into DD:HH:MM:SS string format.
+ * @param totalSeconds The total time in seconds.
+ * @returns Formatted time string.
+ */
+export function formatTime(totalSeconds: number): string {
+  if (totalSeconds < 0) {
+    totalSeconds = 0;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)